2048/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM nginx:latest
+
+RUN apt-get update
+RUN apt-get install -y zip curl
+
+RUN echo "daemon off;" >> /etc/nginx/nginx.conf
+RUN curl -o /usr/share/nginx/html/master.zip -L https://codeload.github.com/gabrielecirulli/2048/zip/master
+WORKDIR  /usr/share/nginx/html/ 
```

Comment:
The whole installation process can be done in a single RUN instruction too.
I don't think WORKDIR makes sense at this point.

You could use other Dockerfiles in this repo as a reference.